src/lib/mathlib/math/Limits.hpp

```diff
@@ -55,29 +55,33 @@ namespace math
 {
 
 template<typename _Tp>
-inline constexpr const _Tp &min(const _Tp &a, const _Tp &b)
+constexpr const _Tp &min(const _Tp &a, const _Tp &b)
 {
 	return (a < b) ? a : b;
 }
 
 template<typename _Tp>
-inline constexpr const _Tp &max(const _Tp &a, const _Tp &b)
+constexpr const _Tp &max(const _Tp &a, const _Tp &b)
 {
 	return (a > b) ? a : b;
 }
 
 template<typename _Tp>
-inline constexpr const _Tp &constrain(const _Tp &val, const _Tp &min_val, const _Tp &max_val)
+constexpr const _Tp &constrain(const _Tp &val, const _Tp &min_val, const _Tp &max_val)
 {
 	return (val < min_val) ? min_val : ((val > max_val) ? max_val : val);
 }
 
-float __EXPORT radians(float degrees);
-
-double __EXPORT radians(double degrees);
-
-float __EXPORT degrees(float radians);
+template<typename T>
+constexpr T radians(const T degrees)
+{
+	return (degrees / static_cast<T>(180)) * static_cast<T>(M_PI);
```

Comment:
Can you change the bracketing, so that the compiler can reduce it to a single multiplication?
`degrees * (static_cast<T>(M_PI) / static_cast<T>(180))`

Reply:
Done